app/views/search/_granule_search.html.erb

```diff
@@ -78,7 +78,7 @@
           <span data-bind="visible: !cmrGranuleQuery.validateCloudCoverRange(cmrGranuleQuery.cloudCover.min(),cmrGranuleQuery.cloudCover.max())" class="error">Minimum must be less than Maximum</span>
         </p>
       </dd>
-      <!-- ko if: $root.labs() && searchable_attributes -->
+      <!-- ko if: $root.labs() && searchable_attributes || tags() && tags()['<%=Rails.configuration.cmr_tag_namespace.gsub(/\.$/, '')%>.features.attribute_search'] && tags()['<%=Rails.configuration.cmr_tag_namespace.gsub(/\.$/, '')%>.features.attribute_search'].data -->
```

Comment:
The &&'s and ||'s are messed up here. Should not show Collection-Specific attributes if there are no searchable_attributes no matter what is contained in labs or tags. Right now, if the collection is tagged, the header shows up regardless of whether there are actually any attributes to show.